presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPruneCountAggregationOverScalar.java

```diff
@@ -28,37 +27,19 @@
 import com.facebook.presto.tpch.TpchTableHandle;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 import static com.facebook.presto.spi.type.DoubleType.DOUBLE;
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.values;
 import static com.facebook.presto.tpch.TpchMetadata.TINY_SCALE_FACTOR;
-import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public class TestPruneCountAggregationOverScalar
-        extends BasePlanTest
+        extends BaseRuleTest
```

Comment:
Same for `TestPushTableWriteThroughUnion`, `TestTransformCorrelatedScalarAggregationToJoin`, `TestTransformExistsApplyToScalarLateralJoin`. Could you fix all of them at once?